pkgs/development/tools/xcbuild/developer.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, platform, toolchain, xcbuild, writeText }:
+
+stdenv.mkDerivation {
+  name = "Xcode.app";
```

Comment:
The name can be changed but the main reason this is here is that xcbuild/xcodebuild are set up to have a "root dir" kind of thing that all other paths are based on. This corresponds to `/Applicatinos/Xcode.app` if you have Xcode installed. We can definitely try to do something more clever with this like setting it to $out/Developer in wrapper.nix and linking `${xcbuild.raw.specs}` or something like that. The last time I tried something like this, though, it seemed to break everything. That was a while back so maybe it is still worth a try now.